package.json

```diff
@@ -47,8 +47,8 @@
     "async": "^0.9.0",
     "bluebird": "^2.9.14",
     "debug": "^2.1.3",
-    "elastic-deletebyquery": "1.0.7",
-    "elasticsearch": "^11.0.1",
+    "elastic-deletebyquery": "^1.0.8",
```

Comment:
No need to upgrade "elastic-deletebyquery", this is true ... I just check the versions and I thought why not used the latest? :) -> you can remove it